frontend/src/components/EntityBulkAdd.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div v-for="(form, index) in entityForms" :key="form.props.id">
+    <div :id="`form-${index}`">
+      <div :class="`${entityForms.length < 2 && 'd-none'} ${index > 0 && 'mt-5 border-top border-light'} row`">
+        <div class="col">
+          <button type="button" :class="`btn-close float-end ${index > 0 ? 'my-3': 'mb-3'}`"
+                  @click="closeForm(form.props.id)"/>
+        </div>
+      </div>
+      <component :is="form.component" @save-all="saveAll" v-bind="form.props"
+                 :ref="el => { refs[`entity-form-${index}`] = el }"/>
+    </div>
+  </div>
+  <div class="container mt-2">
+    <button class="btn btn-outline-secondary w-100" @click="addNewItem">
+      <i class='eos-icons'>add_circle</i>
+      Add more
+    </button>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'EntityBulkAdd'
+}
+</script>
```

Comment:
I'm a bit new to the composition API, but if there is a `<script setup>` tag, this `<script>` tag becomes obsolete, right?

Reply:
> I'm a bit new to the composition API
Yeah, me too and it turned out that we need that, because otherwise the `EntityBulkAdd` was not rendered. 

I found [this thread](https://stackoverflow.com/questions/67669820/how-to-define-component-name-in-vue3-setup-tag) which helped me resolving it..

The strange thing here is that I saw a comment in the thread:

> Update from vue site "Since version 3.2.34, a single-file component using <script setup> will automatically infer its name option based on the filename, removing the need to manually declare the name  

I see that we're using version `3.3.6`, but if we not specify name in separate script tag, it does not work. It could be related with the fact that the rest of the app still uses options API, but IDK..

Reply:
I've stumbled upon the same thing - i.e. the name attribute being removed - but it shouldn't matter if the rest is still options or not, the name should be inferred from the file name (as you mention it) and available as if you'd written it explicitly.

Reply:
isn't the culprint a wrong import? Shouldn't `import EntityBulkAdd from "@/components/EntityBulkAdd.vue";` be without the `.vue` at the end? :thinking:

Reply:
Yes, that's it! Thank you!

Reply:
I should blame the auto import feature from JetBrains for this :laughing:

Reply:
Nicely spotted. I had completely overlooked the `.vue` suffix.